pkg/gardenlet/controller/managedseed/managedseed_controller.go

```diff
@@ -31,9 +31,30 @@ func (c *Controller) managedSeedAdd(obj interface{}) {
 		return
 	}
 
-	if managedSeed.Generation != managedSeed.Status.ObservedGeneration || managedSeed.DeletionTimestamp != nil {
+	generationChanged := managedSeed.Generation != managedSeed.Status.ObservedGeneration
+	jitterUpdates := (c.config.Controllers.ManagedSeed.JitterUpdates != nil) && *(c.config.Controllers.ManagedSeed.JitterUpdates)
+	if managedSeed.DeletionTimestamp != nil {
 		c.logger.Debugf("Added ManagedSeed %s without delay to the queue", key)
 		c.managedSeedQueue.Add(key)
+		return
+	}
+
+	// Newly created managed seed will be enqueued immediatly. Generation is 1 for newly created objects.
+	if managedSeed.Generation == 1 {
+		c.logger.Debugf("Added ManagedSeed %s without delay to the queue", key)
+		c.managedSeedQueue.Add(key)
+		return
+	}
+
+	if generationChanged {
```

Comment:
Are we still able to manually trigger a reconciliation via the operation annotation?
I think we should make that possible to not "lock ourselves out"

Reply:
The existing operation annotation just increases the generation. For `Shoot`s we also don't have an alternative "immediate trigger" in case the `confineSpecUpdates` field is set to `true`. So I think as part of this PR we should be fine. It can still be discussed separately later on if needed.